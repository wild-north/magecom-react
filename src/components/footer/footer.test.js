import React from 'react';
import { Footer } from './index';
import renderer from 'react-test-renderer';

it('Футер отображается корректно', () => {
    const tree = renderer.create(
        <Footer/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});