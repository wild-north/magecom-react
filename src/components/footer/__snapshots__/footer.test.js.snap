// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Футер отображается корректно 1`] = `
<footer
  className="footer footer"
>
  <span>
     © крайний СЕВЕР 2019
  </span>
</footer>
`;
